docs/source/guides/staking_guide.rst

```diff
@@ -0,0 +1,149 @@
+=======================
+NuCypher Staking Guide
+=======================
+
+**Ursula Staking Actions**
+
++-------------------+-------------------------------------------------------------------------+
+| Action            |  Description                                                            |
++===================+=========================================================================+
+|  `stake`          | Initialize or view NuCpher stakes (used with `--value` and `--duration`)|
++-------------------+-------------------------------------------------------------------------+
+|  `divide-stake`   | Divide stake, used with `--value` and `--duration`                      |
++-------------------+-------------------------------------------------------------------------+
+|  `collect-reward` | Collect staking reward (Policy reward collection in future PR)          |
++-------------------+-------------------------------------------------------------------------+
+
+
+**Ursula Staking Options**
+
++----------------+----------------------------------------+
+| Option         |  Description                           |
++================+========================================+
+|  `--value`     | Stake value                            |
++----------------+----------------------------------------+
+|  `--duration`  | Stake duration of extension            |
++----------------+----------------------------------------+
+|  `--index`     | Stake index                            |
++----------------+----------------------------------------+
+|  `--list`      | List stakes (used with `stake` action) |
++----------------+----------------------------------------+
+
+
+
+Interactive Method
+------------------
+
+.. code:: bash
+
+    (nucypher)$ nucypher ursula stake
+
+
+*Initial a new stake*
+
+.. code:: bash
+
+    Stage a new stake? [y/N]: y
+
+    Current balance: 10000
```

Comment:
At the moment, current balance is still unconverted (I think?). E.g. I have `Current balance: 1000000000000000000000000000`.

Reply:
Heh - Yeah I was waiting for someone to notice - I think there are several incorrect conversions, too (`Web.toWei(stake, 'ether')` or is it?).

Perhaps a `class NU` will help?  I can see some kind of denomination handling being a very near PR, or in this one, either way it needs an Issue.

Reply:
following up in PR #830